net/redis/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2007-2015 OpenWrt.org
```

Comment:
I asked @pdf yesterday and he agreed about the porting. The credit of the entire makefile belongs to him, I have just adapted the code to the new sources as said @zorun